### DISCUSSION TITLE
Seeing error in Search setup - no docker startup

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#2627] [Unsupported Versions]
After initial install, seeing error in Search setup - mysql not loading. Components sit in Waiting_Start.

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/5815146/sosetup.log)

Any ideas?

**Comment by juddcbrown** [comment]
Container statuses are now all "Missing".

**Reply by m0duspwnens** [reply]
Can you check the version of the salt-minion running on that node?

```
salt-minion --versions-report
```

Was this an iso install or network install using so-setup-network?

**Comment by juddcbrown** [comment]
[salt_minions.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5815253/salt_minions.txt)

**Comment by juddcbrown** [comment]
.iso install.

**Comment by m0duspwnens** [comment]
Do you know which version of the iso was used? Also, any chance you could download the latest iso and reinstall using it?

**Comment by juddcbrown** [comment]
I just ran # salt 'search-node' state.highstate from manage node. so-redis-count went to zero, all containers are OK.

Still seeing 0 logs in Kibana, but the original issue is fixed.

Thanks!

**Comment by m0duspwnens** [comment]
@juddcbrown 
 The iso was older and had an older version of the salt-minion, 3001.1, and the manager is running 3002.2. Part of the install process for a remote node is to first run the salt state to order to upgrade the salt-minion package if required. 

It looks like in this case, the salt-minion may have still been updating during the setup process, which caused the failure at `23% - GENERATING CA AND CHECKING IN`. Since the minion version wasn't on the version required by the master, it didn't allow the CA state, or any of the preceding state runs with the ID of *_state_not_allowed to be run.

By the time we reached `63% - INSTALLING YUM UTILITIES` in the setup process, the salt-minion had finished upgrading and allowed the yum state and other states to run, but it looks like many failed to pull the images.

The highstate that you ran likely fixed many of the issues. Are you still not seeing any data in Kibana or Hunt? Are the containers still in the OK status? Do you have a forward node that you have confirmed is seeing traffic on the monitor interface?

Thanks,
Josh
